movie.sh

```diff
@@ -24,45 +24,22 @@ do
 done < "$input"
 rm mov.txt
 rm submov.txt
-rate=$(printf %.2f $rating)
+rate=$(printf %.1f $rating)
 echo $rate
 }
 
-
+rm out.txt
 ls movies > movlist.txt
 sed 's/_/./g' movlist.txt > mov2.txt
 input2="mov2.txt"
+echo Rating '     '  Novie >> out.txt
 while IFS= read -r var2
```

Comment:
`var2` is a bad name for a variable. Variable names should explain what they hold. Similarly `i`, `j`, `k` as variable names are also bad.